tests/config/platformFixtures.ts

```diff
@@ -21,11 +21,15 @@ export type PlatformWorkerFixtures = {
   isWindows: boolean;
   isMac: boolean;
   isLinux: boolean;
+  browserPlatform: 'win32' | 'darwin' | 'linux';
+  isRemote: boolean;
 };
 
 export const platformTest = test.extend<{}, PlatformWorkerFixtures>({
   platform: [process.platform as 'win32' | 'darwin' | 'linux', { scope: 'worker' }],
   isWindows: [process.platform === 'win32', { scope: 'worker' }],
   isMac: [process.platform === 'darwin', { scope: 'worker' }],
   isLinux: [process.platform === 'linux', { scope: 'worker' }],
+  browserPlatform: [process.env.PW_TEST_SNAPSHOT_SUFFIX === 'docker' ? 'linux' : process.platform as 'win32' | 'darwin' | 'linux', { scope: 'worker' }],
```

Comment:
I think we can just use `platform` instead, and change it to `linux` when running in docker.